js/.eslintrc.js

```diff
@@ -3,6 +3,7 @@ module.exports = {
     browser: true,
     es2021: true,
   },
+  ignorePatterns: ["dist/"],
```

Comment:
eslint ignore any type was not getting added to .d files during build so declared types with any and eslint ignores were not being ignored after being compiled, don't think we need to lint dist anyways